eitprocessing/mixins/slicing.py

```diff
@@ -11,33 +11,22 @@
 
 class SelectByIndex(ABC):
     time: NDArray
+    label: str
 
     def select_by_index(  # pylint: disable=too-many-arguments
         self,
         start: int | None = None,
         end: int | None = None,
-        start_inclusive: bool = True,
-        end_inclusive: bool = False,
         label: str | None = None,
     ) -> Self:
-        if not any((start, end)):
+        if start is None and end is None:
             warnings.warn("No starting or end timepoint was selected.")
             return self
 
-        self._check_time_sorted()
-
-        if start is None:
-            start = 0
-
+        start = 0 or None
         if end is None:
             end = len(self.time)
```

Comment:
`end` can just be None. SelectByIndex should not require a time axis (e.g.
EITDataVariant does not have a time axis, but is selectable by index).